New formula to install bio++ library

kgori opened the pull request:
Bio++ is a set of C++ libraries for Bioinformatics, including sequence analysis, phylogenetics, molecular evolution and population genetics.

dunn commented on biopp.rb:
```diff
@@ -0,0 +1,68 @@
+    %w[bppcore bppseq bppphyl bpppopgen bppseqomics bppraa].each do |r|
+      resource(r).stage do
+        Dir.mkdir "build"
+        Dir.chdir("build") do
```
These two lines can be replaced with `mkdir "build" do`

steeplejack commented:
This formula should hopefully be OK to merge now.

kgori commented:
Can this be merged?

tdsmith commented on biopp.rb:
```diff
@@ -0,0 +1,57 @@
+    end
+  end
+  test do
+    File.exist? "#{include}/Bpp.all"
```
Can the test build and link a really simple program against the library? Just a few lines from e.g. http://biopp.univ-montp2.fr/wiki/index.php/Alphabet would be great!

kgori replied:
Thanks for the suggestion. It works on my machine, but we'll see what jenkins has to say about it.

tdsmith commented:
Thanks! I squashed your commits and changed the test to use a heredoc instead of string concatenation and then to invoke the compiled binary; the result is in e1fc14b95a. Homebrew thrives because of pull requests like this!

kgori commented:
Great, thanks!
On 4 Jun 2015 17:11, "tim smith" <notifications@github.com> wrote:

> Thanks! I squashed your commits and changed the test to use a heredoc
> instead of string concatenation and then to invoke the compiled binary; the
> result is in e1fc14b
> <https://github.com/Homebrew/homebrew-science/commit/e1fc14b95a7758c8cced3ee447f42e574253816c>.
> Homebrew thrives because of pull requests like this!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew-science/pull/2238#issuecomment-108952928>
> .
>
